### PR TITLE
Update personalCode back button, updated njk

### PR DESCRIPTION
**Pull request by hemerem-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/controllers/personalCodeController.ts [hunk 1]
```diff
@@ -26,3 +28,14 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
...
+export const getPreviousPage = (selectedOption: string): string => {
+    if (selectedOption === "use_name_on_public_register_no") {
+        return BASE_URL + PERSONS_NAME_ON_PUBLIC_REGISTER;
+    } else if (selectedOption === "use_name_on_public_register_yes") {
+        return BASE_URL + USE_NAME_ON_PUBLIC_REGISTER;
+    } else {
+        return BASE_URL + USE_NAME_ON_PUBLIC_REGISTER;
+
+    }
+};
```
Comment (sardhani-ch) [inline src/controllers/personalCodeController.ts:41]: Do you need both else if and else conditions ?? They both are navigating to USE_NAME_ON_PUBLIC_REGISTER

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/controllers/personalCodeController.ts [hunk 2]
```diff
@@ -26,3 +28,12 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
     const lang = selectLang(req.query.lang);
     res.redirect(addLangToUrl(BASE_URL + EMAIL_ADDRESS, lang));
 };
+
+export const getPreviousPage = (selectedOption: string): string => {
```
Comment (ttingle-ch) [inline src/controllers/personalCodeController.ts:32]: If this function is only used on this page it does not need the key word export

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by cbroadley-ch — APPROVED** [review]

**Review by ttingle-ch — APPROVED** [review]

**Pull request merged**